.github/workflows/publish-rc.yml

```diff
@@ -0,0 +1,11 @@
+name: Publish RC to NPM
+on:
+  workflow_dispatch:
+
+jobs:
+  publish:
+    uses: ./.github/workflows/nodejs-setup.yml
+    with:
+      node_version: '18.x'
+      npm_tag: rc
```

Comment:
npm tag goes here; end-users need to explicitly select this tag and it won't "accidentally" be treated as a dependency.